src/ukify/ukify.py

```diff
@@ -4,7 +4,7 @@
 # pylint: disable=missing-docstring,invalid-name,import-outside-toplevel
 # pylint: disable=consider-using-with,unspecified-encoding,line-too-long
 # pylint: disable=too-many-locals,too-many-statements,too-many-return-statements
-# pylint: disable=too-many-branches
+# pylint: disable=too-many-branches,fixme
```

Comment:
hmm?

Reply:
It disables a warning from pylint about 'TODO' and other comments. I use pylint to check syntax and common errors, and those warnings are annoying.